pontoon/sync/tests/test_changeset.py

```diff
@@ -300,3 +299,73 @@ def test_obsolete_db(self):
         self.changeset.execute()
         self.main_db_entity.refresh_from_db()
         assert_true(self.main_db_entity.obsolete)
+
+
+class AuthorsTests(FakeCheckoutTestCase):
+    """
+    Tests authors of translations passed to the final commit message.
+    """
+    def test_multiple_authors(self):
+        """
+        Commit message should include authors from translations of separate
+        entities.
+        """
+        first_author, second_author = UserFactory.create_batch(2)
+        TranslationFactory.create(locale=self.translated_locale, entity=self.main_db_entity,
+            user=first_author, approved=True)
+        TranslationFactory.create(locale=self.translated_locale, entity=self.main_db_entity,
+            approved=False)
+        TranslationFactory.create(locale=self.translated_locale, entity=self.other_db_entity,
+            user=second_author, approved=True)
+        TranslationFactory.create(locale=self.translated_locale, entity=self.other_db_entity,
+            approved=False)
+
+        self.changeset.update_vcs_entity(self.translated_locale.code, self.main_db_entity, MagicMock
+        ())
```

Comment:
Just leave this on the line above, or move the entire MagicMock name to the following line. Splitting it up makes it unclear that MagicMock is being called.